### PR TITLE
Added files via upload

### PR DESCRIPTION
**Pull request by nbarel**
Modal - Nir and Haggai

### CODE REVIEW

File: modals.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!DOCTYPE html5>
```
Comment (krulik) [inline modals.html]: Wrong doctype.

**Comment by krulik** [comment]
@nbarel 
Please refactor class names to follow SUIT CSS naming convention like required.
https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md

File: modals.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  padding-left: 10;
+  padding-top: 5;
+  font-size: 20;
+}
+.modal-close{
+  float: right;
+  padding-right: 10;
+  padding-top: 10;
+}
+a:link {
```
Comment (krulik) [inline modals.css]: You shouldn't style global elements in a component.

File: modals.html [hunk 3]
```diff
@@ -7,6 +7,39 @@
   <link rel="stylesheet" href="modals.css">
 </head>
 <body>
-    
+  <section class="background">
```
Comment (krulik) [inline modals.html]: Not clear why is it needed, is it the overlay of the modal or just a placeholder for body content?

File: modals.html (hunk 1 above)
Comment (nbarel) [inline modals.html]: When I used <!DOCTYPE html> the hole modal gets crap, can you tell me why?

File: modals.css (hunk 2 above)
Comment (nbarel) [inline modals.css]: I used it to eliminate the underline, any other way I can do this?

File: modals.html (hunk 3 above)
Comment (nbarel) [inline modals.html]: overlay, so user wont be able to reach the window behind the modal

Comment (krulik) [inline modals.html]: So the "show modal" link shouldn't be inside it, and I'd like to see how the overlay hides it (with opacity).

**Pull request merged**